snapcraft/plugins/rust.py

```diff
@@ -193,21 +194,74 @@ def _get_target(self) -> str:
             )
         return rust_target.format("unknown-linux", "gnu")
 
+    def _project_uses_workspace(self) -> bool:
+        path = Path(self.builddir, "Cargo.toml")
+        if not path.is_file():
+            return False
+
+        config = open(path).read()
+        return "workspace" in toml.loads(config)
+
+    def _install_workspace_artifacts(self) -> None:
+        """Install workspace artifacts."""
+        # Find artifacts in release directory.
+        release_dir = Path(self.builddir, "target", "release")
+
+        # Install binaries to bin/.
+        bins_dir = Path(self.installdir, "bin")
+        bins_dir.mkdir(parents=True, exist_ok=True)
+
+        # Install shared objects to usr/lib/<arch-triplet>.
+        # TODO: Dynamic library support needs to be properly added.
+        # Although weinstall libraries if we find them, they are most
```

Comment:
missing space typo

Reply:
Hrm, odd that the `codespell` doesn't flag it... will fix